frontend/public/components/sidebars/resource-sidebar-samples.tsx

```diff
@@ -228,6 +232,31 @@ const defaultSamples = ImmutableMap<GroupVersionKind, Sample[]>()
         targetResource: getTargetResource(ConsoleLinkModel),
       },
     ],
+  )
+  .setIn(
+    [referenceForModel(ConsoleModel)],
+    [
+      {
+        title: 'Add catalog categories',
+        description:
+          'Provides a list of default categories which are shown in the Developer Catalog.',
+        id: 'default',
```

Comment:
Perhaps `devcatalog-categories` is a better id. Although I don't know how these ids are used.